2_process/src/observed_data_helpers_prepare.R

```diff
@@ -0,0 +1,29 @@
+unzip_and_split_observed_data <- function(target_name, obs_zipfile, split_file_prefix) {
+  
+  # Unzip the file
+  unzipped_obs_file <- unzip(zipfile = obs_zipfile, overwrite = TRUE, exdir = tempdir())
+  
+  # Read in the csv & split into a feather file per lake
+  local_filenames <- readr::read_csv(unzipped_obs_file, col_types = 'cDddc') %>% 
+    split(.$site_id) %>% 
+    purrr::map(function(data) {
+      split_fn <- sprintf("2_process/tmp/%s_%s.feather", split_file_prefix, unique(data$site_id))
```

Comment:
My own preference would be to define `split_file_prefix` as the whole prefix including file path, i.e., `2_process/tmp/split_obs_data`, so that I could see where the files would get written just by reading the remake file. Not a big deal, though.

Reply:
That makes sense to me! See https://github.com/USGS-R/lake-temperature-out/pull/27/commits/573e1e568f043839014c740986c1dad466856d91